crates/ironclaw_webui_v2_static/static/js/i18n/de.js

```
@@ -657,35 +657,73 @@ registerPack("de", {
   "automations.filterLabel": "Automatisierungsstatusfilter",
   "automations.filter.all": "All",
   "automations.filter.active": "Aktiv",
-  "automations.filter.running": "Running",
-  "automations.filter.failures": "Failures",
+  "automations.filter.running": "Läuft",
```

Biome (2.5.0)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)